data/json/items/ammo.json

```diff
@@ -285,9 +285,11 @@
         "ammo_type" : "BB",
         "damage" : 2,
         "range" : 12,
-        "dispersion" : 11,
-        "count" : 200,
-        "effects" : ["NEVER_MISFIRES"]
+        "dispersion" : 15,
```

Comment:
Might instead be best to pick a new value in line with other ammunition, if a multiplier will produce abnormal results.